data.sql

```diff
@@ -32,4 +32,39 @@ UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='San Smith')
 UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='Jennifer Orwell') WHERE name IN('Gabumon', 'Pikachu');
 UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='Bob') WHERE name IN('Devimon', 'Plantmon');
 UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='Melody Pond') WHERE name IN('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='Dean Winchester') WHERE name IN('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owners_id=(SELECT id FROM owners WHERE full_name='Dean Winchester') WHERE name IN('Angemon', 'Boarmon');
+
+------------------------------------------------------------------------------------------------
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Vet William Tatcher',45,'2000-04-23');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Vet Maisy Smith',26,'2019-01-17');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Vet Stephanie Mendez',64,'1981-05-04');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Vet Jack Harkness',38,'2008-06-08');
```

Comment:
-  [OPTIONAL] I noticed you've been adding entries to the `vets` table by prefixing the names with `vet` such as `Vet Stephanie Mendez`.' Just a quick note that it's not necessary to include `vet`' in the names since it's already the `vets` table. You can simply enter the names as `Stephanie Mende`' or `Jack Harkness` for clarity and consistency. :+1: :100: Remember to also update the names in your `queries.sql` file. :smile: :small_airplane: